atomate2/openmm/jobs/core.py

```diff
@@ -0,0 +1,147 @@
+
+from typing import List, Union, Optional, Dict
+from pathlib import Path
+
+from dataclasses import dataclass
+
+from jobflow import Maker
+
+from pymatgen.io.openmm.schema import InputMoleculeSpec
+from pymatgen.io.openmm.sets import OpenMMSet
+
+from atomate2.openmm.jobs.base import BaseOpenmmMaker
+
+
+@dataclass
+class InputMaker(Maker):
+    name: str = "input maker"
+
+    def make(
+        self,
+        input_mol_dicts: List[Union[Dict, InputMoleculeSpec]],
+        density: Optional[float] = None,
+        box: Optional[List[float]] = None,
+        prev_dir: str | Path | None = None
+    ):
+        """
+        Create a job that generates an OpenMM input set.
+
+        Parameters
+        ----------
+        input_set : .OpenMMSet
+            A pymatgen structure object.
+        density : float
```

Comment:
What are the units of density mean in this context?

Reply:
The pymatgen.io.openmm [tutorial](https://github.com/orionarcher/pymatgen-io-openmm/blob/main/tutorials/basic_setup_tutorial.ipynb) says `density of solution` and after poking through the source I see [get_box](https://github.com/orionarcher/pymatgen-io-openmm/blob/9801016f7cb2a4bad38ef35fb52797f42ee7f89b/pymatgen/io/openmm/utils.py#L88) saying it's the density of molecules per cubic Angstrom. Do I get that right?

Reply:
It's the density in g/ml, I should specify the units somewhere. That's the unit scale most people are used to.